docs/introduction-to-smart-contracts.rst

```diff
@@ -43,7 +43,7 @@ blockchain. The line ``uint storedData;`` declares a state variable called ``sto
 type ``uint`` (*u*\nsigned *int*\eger of *256* bits). You can think of it as a single slot
 in a database that you can query and alter by calling functions of the
 code that manages the database. In the case of Ethereum, this is always the owning
```

Comment:
Maybe it is easier to understand if we just remove the sentence 'In the case of Ethereum, this is always the owning contract.`?

Reply:
Personally I feel it should be left in. This is the introduction, maybe no one knows what Ethereum is yet. We make a comparison above to a database, which many developers understand, but then we say what the equivalent of that is with Ethereum.

Reply:
I agree, but the paragraph does the comparison. Then we explain which functions in ethereum can modify the value.

Reply:
OK, removed, but added an additional few words after.